eng/Packages.props

```diff
@@ -23,7 +23,7 @@
     <PackageReference Update="System.Net.Http" Version="4.3.4" />
     <PackageReference Update="System.Memory" Version="4.5.4" />
     <PackageReference Update="System.Reflection.Metadata" Version="1.6.0" />
-    <PackageReference Update="System.Resources.Extensions" Version="$(SystemResourcesExtensionsPackageVersion)" />
```

Comment:
why this change? Should we just update `SystemResourcesExtensionsPackageVersion` to `4.6.0`?

Reply:
reverted this. SystemResourcesExtensionsPackageVersion was not being defined on my machine and this change accidentally got pushed